stable/elasticsearch/templates/configmap.yaml

```diff
@@ -45,6 +45,11 @@ data:
     xpack.monitoring.enabled: ${XPACK_MONITORING_ENABLED:false}
     xpack.security.enabled: ${XPACK_SECURITY_ENABLED:false}
     xpack.watcher.enabled: ${XPACK_WATCHER_ENABLED:false}
+{{- else }}
+    xpack.ml.enabled: false
```

Comment:
This feature was released in 5.4 and will fail in 5.3 and 5.2

Reply:
I've tested this PR with 5.6, and it works. @cameronattard, are you seeing failure with 5.2 and 5.3 as @tarasovvl suggested?

Reply:
I haven't tested, but if it is broken in this PR, that likely means it's also broken in the current release. Fixing that is out of scope for this PR.

Reply:
since the feature is related to es5, it would be nice to fix for all 5.x versions and not introduce new issues.
could you wrap xpack.ml.enabled: false with condition @anapsix suggested?

Reply:
@tarasovvl I've addressed this issue - care to test it out?